controller/array_action/array_mediator_interface.py

```diff
@@ -266,19 +266,20 @@ def delete_snapshot(self, snapshot_id):
         raise NotImplementedError
 
     @abstractmethod
-    def get_iscsi_targets_by_iqn(self):
+    def get_iscsi_targets_by_iqn(self, host_name=None):
```

Comment:
actually I think it would be simpler to avoid default values in the interface, because the caller would also pass it

Reply:
same for [existing method](https://github.com/IBM/ibm-block-csi-driver/blob/a91fb4de07da7c95d94be308c3d1006360fc6d9d/controller/array_action/array_mediator_interface.py#L287)s
![image](https://user-images.githubusercontent.com/57443811/131718335-25cb6e8d-b345-492a-9997-15a2749e6f17.png)

Reply:
done, please open a ticket for non-scoped issues..

Reply:
for existing gaps of this type, we already have [this](https://github.com/IBM/ibm-block-csi-driver/blob/8a2506fc14bde06a63937c31a7e135596c3642b3/controller/scripts/lint.ini#L30), so no need for a separate ticket here